MCprep_addon/materials/generate.py

```diff
@@ -29,6 +30,18 @@
 
 AnimatedTex = Dict[str, int]
 
+class PackFormat(Enum):
+	def _generate_next_value_(name, start, count, last_values):
+		return name
+	
+	SIMPLE = auto()
+	SEUS = auto()
+	SPECULAR = auto()
+  
+	@staticmethod
+	def from_str(enum_name: str):
+			return PackFormat[enum_name]
```

Comment:
This isn't needed, the behavior of `Enum[string]` is already documented or could be inferred.